test/integration/roles/test_amazon_rds/tasks/main.yml

```diff
@@ -0,0 +1,155 @@
+# create and kill an RDS
+
+# Improvements expected in later versions of this test case
+
+# use present instead of create
+# use absent instead of delete
+# use SSD
+
+
+# Start with creation - N.B. if the RDS already exists from a previous run this
+# may not change anything so we don't insist on that
+
+  - name: create RDS
+    tags: test_amazon_rds
+    local_action:
```

Comment:
Should be `rds:`

You can always use `connection: local` at a task level but I'd expect the playbook to take care of where the task is run.